### PR TITLE
BUG: Copy to xarray by default

### PR DESCRIPTION
**Pull request by tbirdso**
Previously observed behavior where ITK data was released after ITK
Python filter went out of scope, resulting in cascading and difficult to
trace fatal memory r/w error on xarray access. This change makes data
copy the default behavior while exposing `in_place` option for cases
where operations are contained within a single block and behavior
benefits from in-place operations.

Addresses discussion in [https://github.com/spatial-image/multiscale-spatial-image/issues/44](https://github.com/spatial-image/multiscale-spatial-image/issues/44)

## PR Checklist
- [x] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [x] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)

Refer to the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) for
further development details if necessary.

### CODE REVIEW

File: Wrapping/Generators/Python/itk/support/xarray.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 ]
 
 
-def xarray_from_image(l_image: "itkt.ImageOrImageSource") -> "xr.DataArray":
+def xarray_from_image(l_image: "itkt.ImageOrImageSource", in_place:bool=False) -> "xr.DataArray":
```
Comment (thewtex) [inline Wrapping/Generators/Python/itk/support/xarray.py:37]: A more conventional name would be `view` here

**Review by thewtex — COMMENTED** [review]
@tbirdso excellent!

Naming improvement suggested inline

File: Wrapping/Generators/Python/itk/support/xarray.py (hunk 1 above)
Comment (tbirdso) [reply Wrapping/Generators/Python/itk/support/xarray.py]: Fixed

**Comment by tbirdso** [comment]
Added a short test as well to verify that the "view" is valid in the same block.

**Review by thewtex — APPROVED** [review]
:100:

**Pull request merged**